In [22]:
import pandas as pd
from astropy import units as u
from astropy.coordinates import SkyCoord
import numpy as np

In [23]:
data = pd.read_csv('Outliers.csv')

In [24]:
runcat, ra, dec = data['runcat'],data['ra'],data['dec']

In [25]:
results = []

In [26]:
for i, val in enumerate(ra):
    coordinates = SkyCoord(ra= val*u.degree, dec=dec[i]*u.degree)
    results.append([runcat[i],coordinates.to_string('hmsdms')])

In [28]:
results = np.array(results)
print(results)

[['485' '18h18m10.50154289s +07d17m25.46915611s']
 ['97' '18h20m30.07813164s +06d34m19.45450388s']
 ['78' '18h20m54.4061415s +07d16m57.93073015s']
 ['66' '18h21m13.18828283s +07d24m37.85915601s']
 ['108' '18h20m21.92707487s +07d11m08.30319846s']
 ['164' '18h18m49.74694386s +06d28m44.63237573s']]


485: NOTHING THERE
97: NVSS J18...
78: OLD 186 
66: ONE 0F THE LOBES OF THE BLACK HOLES 
108: MAXI 
164: NVSSS J18... 



In [11]:
coordinates = SkyCoord(ra= ra[0]*u.degree, dec=dec*u.degree)

In [13]:
print(coordinates)
coordinates.to_string('hmsdms')

<SkyCoord (ICRS): (ra, dec) in deg
    (274.54375643, 7.2904081)>


'18h18m10.50154289s +07d17m25.46915611s'

In [39]:
ra_155, dec_155 = 275.166, 7.145

data = pd.read_csv('Wholedatasetoutput.csv')
runcat, ra, dec = data['runcat'],data['ra'],data['dec']

In [55]:
for i, val in enumerate(ra):
    coordinates = SkyCoord(ra= val*u.degree, dec=dec[i]*u.degree)
    #print(round(coordinates.ra.degree,3))
    if round(coordinates.ra.degree,3) == ra_155 and round(coordinates.dec.degree,3) == dec_155:
        print(runcat[i])

91
